### PR TITLE
Update activity

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/components/ResourceCard/AddResource/Editors/H5PEditor.js [hunk 1]
```diff
@@ -53,12 +53,14 @@ const H5PEditor = (props) => {
...
       };
+
       handleCreateResourceSubmit(
-        match.params.playlistid,
+        match.params.playlistId,
         resource.newResource.activity.h5pLib,
         resource.newResource.activity.h5pLibType,
         payload,
         resource.newResource.metadata,
+        match.params.projectId,
```
Comment (simonebelotti75) [inline src/components/ResourceCard/AddResource/Editors/H5PEditor.js:63]: Do we need projectId to create activity?

File: src/components/ResourceCard/AddResource/style.scss [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+     .methods .radio-inline{
+      padding-right: 10px;
+      cursor: pointer;
+     }
+     
+     .methods .radio-inline input{
+      margin-right: 2px;
+     }
+  }
```
Comment (simonebelotti75) [inline src/components/ResourceCard/AddResource/style.scss:16]: It seems like this part is not correct indented. It should throw eslint error.

File: src/store/actions/resource.js [hunk 3]
```diff
@@ -346,42 +346,38 @@ export const createResourceAction = (
...
+        h5p_content_id: resource.id,
+        playlist_id: playlistId,
+        thumb_url: metadata.thumbUrl,
+        action: 'create',
+        title: metadata.metaContent.metaTitle,
+        type: 'h5p',
+        content: 'place_holder',
+        subject_id:
         metadata.metaContent.metaSubject
         && metadata.metaContent.metaSubject.subject,
```
Comment (simonebelotti75) [inline src/store/actions/resource.js:362]: Indent issue

File: src/store/actions/resource.js [hunk 4]
```diff
@@ -346,42 +346,38 @@ export const createResourceAction = (
...
-      education_level_id:
+        education_level_id:
         metadata.metaContent.metaEducationalLevels
         && metadata.metaContent.metaEducationalLevels.name,
```
Comment (simonebelotti75) [inline src/store/actions/resource.js:365]: Should be indented 2 more spaces

File: src/store/actions/resource.js [hunk 5]
```diff
@@ -346,42 +346,38 @@ export const createResourceAction = (
...
-    dispatch(createResource(playlistId, resource, editor, editorType));
-    // dispatch(hideCreateResourceModal());
-    window.location.href = `/project/${projectId}`;
-  } else {
-    dispatch(validationErrorsResource());
+      };
+      const insertedResource = await resourceService.createActivity(createActivityDate);
+
+      resource.id = insertedResource.id;
+      resource.mysqlid = insertedResource.mysqlid;
```
Comment (simonebelotti75) [inline src/store/actions/resource.js:370]: Do we still need mysqlid?

File: src/store/actions/resource.js [hunk 6]
```diff
@@ -390,48 +386,40 @@ export const createResourceByH5PUploadAction = (
...
+        education_level_id:
+          metadata.metaContent.metaEducationalLevels
+          && metadata.metaContent.metaEducationalLevels.name,
+      };
+
+      const responseActivity = await resourceService.createActivity(createActivityUpload);
+
+      const resource = { ...responseActivity };
+      resource.id = responseActivity.activity.id;
+      resource.mysqlid = responseActivity.mysqlid;
```
Comment (simonebelotti75) [inline src/store/actions/resource.js:419]: Same here

File: src/components/ResourceCard/EditResource/index.js [hunk 7]
```diff
@@ -20,7 +20,8 @@ const EditResource = (props) => {
   const {
     match,
     resource,
-    //showDescribeActivity,
+    update_activity,
```
Comment (simonebelotti75) [inline src/components/ResourceCard/EditResource/index.js:23]: What is this one?

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/store/actions/resource.js (hunk 4 above)
Comment (muhammadqamar) [reply src/store/actions/resource.js:365]: client didn't through error on it. seems to be ok then

File: src/store/actions/resource.js (hunk 3 above)
Comment (muhammadqamar) [reply src/store/actions/resource.js:362]: eslint didn't through error on it. seems to be ok then

File: src/components/ResourceCard/AddResource/style.scss (hunk 2 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/AddResource/style.scss:16]: eslint didn't through error on it. seems to be ok then

File: src/store/actions/resource.js (hunk 6 above)
Comment (muhammadqamar) [reply src/store/actions/resource.js:419]: removed

File: src/components/ResourceCard/EditResource/index.js (hunk 7 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/EditResource/index.js:23]: removed unused data

File: src/store/actions/resource.js (hunk 5 above)
Comment (muhammadqamar) [reply src/store/actions/resource.js:370]: removed

File: src/components/ResourceCard/AddResource/Editors/H5PEditor.js (hunk 1 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/AddResource/Editors/H5PEditor.js:63]: this is for redirecting to main page

**Review by muhammadqamar — COMMENTED** [review]
kindly check

File: src/components/ResourceCard/AddResource/style.scss (hunk 2 above)
Comment (simonebelotti75) [reply src/components/ResourceCard/AddResource/style.scss:16]: It really strange that Eslint errors are not throwing on your local
I will add some script and please run that command before make PR in future

File: src/components/ResourceCard/EditResource/index.js [hunk 8]
```diff
@@ -20,14 +20,13 @@ const EditResource = (props) => {
...
     resource,
-    //showDescribeActivity,
     showResourceDescribeActivity,
     handleHideCreateResourceModal,
   } = props;
 
   useEffect(() => {
     showResourceDescribeActivity(resource, match.params.activityId);
-  }, []);
+  },[]);
```
Comment (simonebelotti75) [inline src/components/ResourceCard/EditResource/index.js:29]: Space should be there

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/components/ResourceCard/EditResource/index.js (hunk 8 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/EditResource/index.js]: added

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**